bundles/GenericExecutionEngineBundle/src/Configuration/ExecutionContextInterface.php

```diff
@@ -19,4 +19,6 @@
 interface ExecutionContextInterface
 {
     public function getTranslationDomain(string $context): string;
+
+    public function getErrorHandlingFromContext(string $context): ?string;
```

Comment:
hmm, is that a BC break? 
Or should that interface be @internal anyway?

Reply:
I don´t think this would be a BC break, its just adding an option for users to override the default error handling behavior on a job run level. The default behavior is still used as a fallback and nothing changes there. 
We can make the interface internal, that's a good point

Reply:
11.3. is not released yet anyway, so no BC break